tests/log.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"io"
+	"os"
+
+	"go.uber.org/zap/zapcore"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+)
+
+// Define a simple encoder config appropriate for testing
+var simpleEncoderConfig = zapcore.EncoderConfig{
+	TimeKey:       "",
+	LevelKey:      "level",
+	NameKey:       "",
+	CallerKey:     "",
```

Comment:
I'm open to us omitting a few of these, but shouldn't we keep `Time`? Seems like timestamps and caller would be helpful for debugging.